src/citrine/_session.py

```diff
@@ -174,6 +174,14 @@ def get_resource(self, path: str, **kwargs) -> dict:
         response = self.checked_get(path, **kwargs)
         return self._extract_response_json(path, response)
 
+    def get_resource_post(self, path: str, json: dict, **kwargs) -> dict:
```

Comment:
What is the difference between this and `post_resource`? Is this just for a more intuitive method name to use when resource retrieval is a POST?

Reply:
I just hadn't noticed that the content was the same as `post_resource`.  While there may be value in having different names, I think simplicity wins here.  I removed `get_resource_post` and am calling `post_resource` in search now.